.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cSpell.words": [
```

Comment:
We should remove this file completely.

Reply:
yes! thank you! i was trying to add "devfile" to my vs code dictionary and it created that file. :shrug: